### DISCUSSION TITLE
Vivado Power Analysis Low Confidence Level

### DISCUSSION THREAD

**Discussion by Ba1tu3han** [Xilinx/finn#1184] [Q&A] [answered]
I compiled the CNV model succesfuly and imported the _**finn_vivado_stitch_proj.xpr**_ file to Vivado 2022.1. In the power analysis, the result is low confidence. And the results does not make sense.

Additionally, I'm very rookie about Vivado.  

What do you suggest to fix it?

![image](https://github.com/user-attachments/assets/c188c799-e947-4246-87ac-213c00a92fd0)

![image](https://github.com/user-attachments/assets/17916a62-924e-40d8-89e6-931ab4d5f596)

Bests,
Batuhan

**Comment by Ba1tu3han** [comment]
Hello FINN Team,

Could you please share a starting point to fix this issue? 

Bests,

@maltanar @auphelia @fpjentzsch

**Comment by auphelia** [comment, marked as answer]
Hi @Ba1tu3han,
Thanks for your message. 
This is not directly a FINN specific topic, so I would refer you to the documentation about the Power Analysis Tool in Vivado: https://docs.amd.com/r/en-US/ug907-vivado-power-analysis-optimization/Power-Analysis-and-Optimization-in-the-Vivado-Design-Suite
I also recommend you consult the Vivado support forums.

**Reply by Ba1tu3han** [reply]
Hello @auphelia , 

Thank you for your reply. It is helpful. 

Bests,
